internal/configuration/validator/access_control.go

```diff
@@ -68,61 +68,76 @@ func ValidateAccessControl(configuration schema.AccessControlConfiguration, vali
 
 // ValidateRules validates an ACL Rule configuration.
 func ValidateRules(configuration schema.AccessControlConfiguration, validator *schema.StructValidator) {
-	for _, r := range configuration.Rules {
-		if len(r.Domains) == 0 {
-			validator.Push(fmt.Errorf("No access control rules have been defined"))
+	if len(configuration.Rules) == 0 {
+		defaultPolicy := strings.ToLower(configuration.DefaultPolicy)
```

Comment:
I'd rather not open the door to case insensitive enums (like aLLoW or deNy or whatever) in the config but rather throw an error if it does not exactly match. It should not be our problem if people don't care about case sensitivity imo. Let's keep control on that. What do you think?

Reply:
Sounds fine to me.